Framework/DataObjects/src/EventList.cpp

```diff
@@ -2924,9 +2924,7 @@ double EventList::getTofMin() const {
  */
 double EventList::getTofMax() const {
   // set up as the minimum available double
-  double tMax =
-      -1. *
-      std::numeric_limits<double>::max(); // min is a small number, not negative
+  double tMax = std::numeric_limits<double>::lowest();
```

Comment:
Am I correct in saying `tof()` just actually means value on X axis? If so can they be negative, unit=dE, for example? If all X is negative then this would not be compatible with the test below?

Reply:
You are correct on the interpretation of `tof`. `std::numeric_limits<double>::lowest()` is the compiler value for most negative number available of the chosen type. From my quick test on gcc7, `std::numeric_limits<double>::lowest() == -1. *
      std::numeric_limits<double>::max()`. Any "reasonable" value of x should still be larger than the most negative value available. Similarly, they should all be less than `std::numeric_limits<double>::max()`.

Reply:
Thanks for the information. I guessed incorrectly that `lowest` was positive - thanks for clearing it up.